pkg/providers/vsphere/config/template.yaml

```diff
@@ -170,7 +170,11 @@ spec:
       apiServer:
         extraArgs:
           cloud-provider: external
+{{- if (eq .format "bottlerocket") }}
```

Comment:
I dont think the cli is the right place to be doing this since we are already handling this path translation stuff in capi to support BR.  That said, leave this at /etc since its the mountPath inside the pod.

Reply:
Gotcha, makes sense